test/importPipeline.js

```diff
@@ -15,11 +15,6 @@ tape('functional test of importing two small OA files', function(t) {
   var expected = JSON.parse(fs.readFileSync(expectedPath));
 
   var endStream = event_stream.writeArray(function(err, results) {
-    // calling toJSON on a Document writes the phrase, so we have to emulate that here
```

Comment:
Hmm, I don't remember how this ended up in this PR. The code isn't needed any more since pelias-model now handles setting phrase. I'll move it to a separate PR.